package/entrypoint.sh

```diff
@@ -6,6 +6,14 @@ if [ ! -e /run/secrets/kubernetes.io/serviceaccount ] && [ ! -e /dev/kmsg ]; the
     exit 1
 fi
 rm -f /var/lib/rancher/k3s/server/cred/node-passwd
+if [ -e /var/lib/rancher/etcd ] && [ ! -e /var/lib/rancher/k3s/server/db/etcd ]; then
+  mkdir -p /var/lib/rancher/k3s/server/db
+  ln -sf /var/lib/rancher/etcd /var/lib/rancher/k3s/server/db/etcd
+  echo -n 'default' > /var/lib/rancher/k3s/server/db/etcd/name
+fi
+if [ -e /var/lib/rancher/k3s/server/db/etcd ]; then
+  k3s server --cluster-reset &> ./k3s-cluster-reset.log
```

Comment:
Note for other reviewers: this is necessary because K3s's embedded etcd can't be bound to 127.0.0.1, and will get mad if the Docker container IP address changes. This `--cluster-reset` doesn't drop any data from the etcd datastore; it just resets the cluster membership list and ensures that the sole etcd cluster member has the correct IP address.